tests/mocha/workspace_svg_test.js

```diff
@@ -93,7 +93,7 @@ suite('WorkspaceSvg', function() {
         this.workspace.getTopBlocks()[0].getInput('NAME').connection;
     inputConnection.connect(block.outputConnection);
     chai.assert.exists(block.getSvgRoot());
-    chai.assert.notExists(shadowBlock.getSvgRoot());
```

Comment:
Is this test specifically about disposing of SVGs, and there's another test that covers other parts of destruction? If so, probably just delete this test instead of modifying. If not, can you change the title to `Replacing shadow disposes of old shadow` + update the `exists(block.getSvgRoot())` assert to `isFalse(block.isDeadOrDying())`?